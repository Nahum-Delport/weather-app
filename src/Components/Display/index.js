import React from 'react';

class Display extends React.Component {
    render(){
        return(
            <div>
                <h1>Weather from Wherever</h1>
                <p>Any place, anywhere.</p>
            </div>
            
        );
    }
}
export default Display;