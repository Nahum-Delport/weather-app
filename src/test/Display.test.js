import React from 'react';
import Display from '../Components/Display/index.js';
import renderer from 'react-test-renderer';

test("renders correctly", () => {
	const tree = renderer.create(<Display />)
	expect(tree).toMatchSnapshot();
});