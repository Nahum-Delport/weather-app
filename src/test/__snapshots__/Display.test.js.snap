// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Weather from Wherever
  </h1>
  <p>
    Any place, anywhere.
  </p>
</div>
`;
